tests/admin_checks/tests.py

```diff
@@ -43,6 +45,11 @@ def check(self, **kwargs):
 )
 class SystemChecksTestCase(SimpleTestCase):
 
+    def setUp(self):
+        super(SystemChecksTestCase, self).setUp()
+        admin.sites.all_sites.clear()
```

Comment:
Adam, I'm unsure what you mean -- are you saying you don't think the test is adding value or that the setUp/tearDown is obsoleted by some change in your PR? I understand this PR is basically the same as what you did in 22f027e1bdce3d74ae828710eeaeca78b8c0c170 (with a test added).